Content.Server/Chat/ChatManager.cs

```diff
@@ -71,15 +75,27 @@ public void EntitySay(IEntity source, string message)
             var pos = source.Transform.GridPosition;
             var clients = _playerManager.GetPlayersInRange(pos, VoiceRange).Select(p => p.ConnectedClient);
 
+            if (message.StartsWith(';') && source.TryGetComponent<InventoryComponent>(out InventoryComponent inventory))
+            {
+                if (inventory.TryGetSlotItem<ItemComponent>(EquipmentSlotDefines.Slots.EARS, out ItemComponent item))
+                {
+                    if (item.Owner.TryGetComponent<HeadsetComponent>(out HeadsetComponent headset))
+                    {
+                        message = message.Substring(1);
```

Comment:
In SS13 the ; will also be removed from the start of the message if the player has no headset on. I am not sure if a lot of stations that would need a message to be started with ; as most of them don't allow smileys anyways.